browse/actagram.c

```diff
@@ -15,6 +15,55 @@
 
 #include "view.h"
 
+
+int is_valid_day(int day)
+{
+    return day >=1 && day <= 31;
+}
+
+int is_valid_month(int month)
+{
+    return month >=1 && month <= 12;
+}
+
+int is_valid_year(int year)
+{
+    return year >=10 && year <= 99;
+}
+
+int is_valid_date(int date)
+{
+    return is_valid_day(date % 100)
+        && is_valid_month((date / 100) % 100)
+        && is_valid_year((date / 10000) % 100);
+}
```

Comment:
It seems kind cumbersome to have a function dedicated to checking if a number is between 1 and 12 etc.  Would this not make MUCH more sense if it was inlined instead?
Yes you could make these functions more sophisticated later, but THAT would be the time to make it into a function, not earlier.

Reply:
I was thinking the same thing but happy to get the feedback.

I was considering pulling `read_holiday_config()` into a new file `browse_config.c`. Since it is only used in `view.c`. But considering the above feedback, maybe you wouldn't want that just yet?